### DISCUSSION TITLE
Question: Memory Consumption

### DISCUSSION THREAD

**Discussion by The-Futurist** [icsharpcode/SharpZipLib#745] [Q&A]
We use version 1.2.0 against .Net F 4.52 and I wanted to ask has there been any significant changes since then that might reduce memory use? We often automatically unzip large CSV files, perhaps 11 or 12 million records. Everything works but we see the app's working set ramp like 5, 6 or more GB during this time so I wanted to get an idea if there might be value in updating before I actually try.

Thanks

**Comment by piksel** [comment]
Not for .NET Framework, no. The main tools for making such optimizations are only present in .NET Core, and there has only been a few of such changes in our code base to take advantage of that even so.
